docs/src/main/sphinx/sql/update.rst

```diff
@@ -34,6 +34,13 @@ Update the account manager and account assign date for all customers::
       account_manager = 'John Henry',
       assign_date = DATE '2007-01-01';
 
+Update the manager to be the name of the employee who matches the manager ID::
```

Comment:
explicitly call out that you can use an expression including a subquery for a column value ... and have an example for each

Reply:
When you say an example for each, what do you mean? This _is_ an example for an expression that includes a subquery, and I don't feel like this page is the proper place to define what an expression is

Reply:
Add another example that uses a simple expression .. maybe a last_modified field that is updated with a datetime stamp related function call..